rst_files/tools_editors.rst

```diff
@@ -17,9 +17,35 @@ We assume you've already completed the :ref:`getting started <getting_started>`
 Preliminary Setup 
 ====================
 
-We strongly suggest following the instructions to create a ``startup.jl`` (see the `getting started <jl_startup_file>`_  for more details)
-* Ensure you have key packages installed ``] add IJulia InstantiateFromURL Revise REPL``
-* Download the file `startup.jl </_static/includes/startup.jl>`_ into  ``~/.julia/config/``
+Follow the instructions for setting up Julia `on your local computer <jl_jupyterlocal>`_
+
+.. _jl_startup_file:
+
+Creating a Startup File (Recommended)
+----------------------------------------------------
+
+
+Whenever the Julia compiler or REPL starts, it will look for a file called ``startup.jl`` (see `Julia Manual <https://docs.julialang.org/en/v1/manual/getting-started/#man-getting-started-1>`_)
+
+The location for the file is relative to your default Julia environment (e.g. ``~/.julia/config/startup.jl`` or ``C:\Users\USERNAME\.julia\config\startup.jl`` on Windows)
```

Comment:
Just added in a code-block here to `DEPOT_PATH[1]`, which will give them the exact path on their machine.